app/frontend/src/styles/application/jobseekers/jobseeker.scss

```diff
@@ -11,9 +11,17 @@
         }
       }
     }
-    
+
     .govuk-main-wrapper__notification {
       background-color: govuk-colour('light-grey');
     }
   }
 }
+
+.govuk-heading-xl.subscription_title {
+  margin-bottom: govuk-spacing(4);
```

Comment:
Instead of creating a class, you could use the override classes: https://design-system.service.gov.uk/styles/spacing/

Reply:
As those classes have unusual characters in them, you'll need to apply the class using 
```ruby
{ class: 'unusual-class-name!' }
```

Reply:
Will take a look. Thank you.

Reply:
These classes don't need to be here

Reply:
Yep sorry, forgot to take them out.